seatunnel-transforms/seatunnel-transforms-spark/seatunnel-transform-spark-encrypt-phone/src/main/resources/META-INF/services/org.apache.seatunnel.spark.BaseSparkTransform

```diff
@@ -0,0 +1 @@
+org.apache.seatunnel.spark.transform.EncryptedPhone
```

Comment:
Please add Apache License header, and add one empty end line.

Reply:
Need to re request？

Reply:
Just push in `wsyhj:dev` branch

Reply:
done